packages/collector/test/hooks.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * (c) Copyright IBM Corp. 2021
+ * (c) Copyright Instana Inc. and contributors 2020
```

Comment:
Want to change to 2022, because you touched it?
Ignore if you don't want to :)

Reply:
I'm also always a bit unsure when exactly that would need to happen, but I think in this case it is more correct to leave the years as it is. In general, the copyright year would be the year when the original work has been created. True, this is a new file, but I basically renamed an existing file, so it is more a modification of the existing work from 2020. And you would not update the copyright year for any modifications of existing work. (See https://stackoverflow.com/a/2391555/2565264 for more information on this very boring topic :-) )

Reply:
I am also not 100% sure. Ok then we agree that changing the copyright year has no use cases.